pom.xml

```diff
@@ -31,7 +31,7 @@
     <scm>
         <connection>scm:git:https://gitbox.apache.org/repos/asf/cxf.git</connection>
         <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/cxf.git</developerConnection>
-        <url>https://gitbox.apache.org/repos/asf?p=cxf.git;a=summary</url>
+        <url>https://gitbox.apache.org/repos/asf/cxf.git</url>
```

Comment:
I think `;a=summary` is confusing, could you please change it to:

```suggestion
        <url>https://gitbox.apache.org/repos/asf?p=cxf.git</url>
```

Thank you.

Reply:
I obviously don't mind doing the change, but I'm wondering why the URL is slightly different on https://gitbox.apache.org/repos/asf:

![image](https://github.com/apache/cxf/assets/1470270/01aa96ba-c00c-4816-a89c-882c8aacb646)

Reply:
And interestingly, if I go to https://gitbox.apache.org/repos/asf?p=cxf.git;a=summary today, I get redirected to https://github.com/apache/cxf/commit/main.
Whereas a couple of days ago, it returned a 404 (as shown in my initial message).

Reply:
I don't know the history, but I suspect the `https://gitbox.apache.org/repos/asf?p=cxf.git` may (in future) redirect to any other place if the Git coordinate changes.